blogs/vonage-solaria-python-fastapi/src/server.py

```
@@ -0,0 +1,449 @@
+        delay = min(MAX_RETRY_DELAY, INITIAL_RETRY_DELAY * (2 ** (gladia_session["retry_count"] - MAX_RETRIES)))
+        # Add jitter
+        delay = delay * (0.5 + random.random())
+        logger.warning(f"Too many Gladia session creation attempts. Backing off for {delay:.2f} seconds")
+        time.sleep(delay)
+    
+    # Try all possible API formats that might work with Gladia
+    # Format 1: Current v2 API format
+    payload = {
+        "sample_rate": 8000,
+        "encoding": "wav/pcm",
+        "sample_rate": 8000,
```

Ruff (0.8.2)

Dictionary key literal `"sample_rate"` repeated

Remove repeated key literal `"sample_rate"`

(F601)